scripts/prepare-for-release

```diff
@@ -123,18 +172,27 @@ create_release_branch() {
 update_versions() {
     # update release version for release prep
     echo -e "🥑 Attempting to update NTH release version in preparation for a new release."
+    UPDATED_CHART_VERSION=$(increment_chart $CURR_CHART_VERSION)
 
     for f in "${FILES[@]}"; do
         # do not exit if grep doesn't find $PREVIOUS_VERSION; continue to exit on all other exit codes
         has_incorrect_version=$(cat $f | grep $PREVIOUS_VERSION || [[ $? == 1 ]])
-        if [[ ! -z  $has_incorrect_version ]]; then
-            sed -i '' "s/$PREVIOUS_VERSION/$LATEST_VERSION/g" $f
+        chart_should_increment=$(cat $f | grep $CURR_CHART_VERSION || [[ $? == 1 ]])
+        if [[ ! -z  $has_incorrect_version ]] || [[ ! -z  $chart_should_increment ]]; then
+            if [[ "${os}" = "Linux" ]]; then
```

Comment:
windows? 😂

Reply:
I seriously debated cloning simple-ec2 repo, build, launch, connect, clone nth repo, make release-prep. But there was way too much setup to run all those commands on Windows D:

either way-- good catch on an opportunity for not knowing wtf is going on. Adding some more messaging